collectors/ioping.plugin/metadata.yaml

```diff
@@ -53,23 +53,24 @@ setup:
           description: Data collection frequency.
           default_value: 1s
           required: false
-        - name: ioping
-          description: The complete plugin path.
-          default_value: /usr/libexec/netdata/plugins.d/ioping
+        - name: destination
+          description: The directory/file/device to ioping.
+          default_value: ""
           required: false
         - name: request_size
           description: The request size in bytes to ioping the destination (symbolic modifiers are supported)
           default_value: 4k
           required: false
-        - name: ioping_options
+        - name: ioping_opts
           description: Options passed to `ioping` commands.
           default_value: -T 1000000
           required: false
     examples:
       folding:
         enabled: true
         title: ''
-      list: []
+      list: |
+        destination="/dev/sda"
```

Comment:
@thiagoftsm please use the schema in order to fill this in, this is not the way we define examples, check https://github.com/netdata/netdata/blob/master/collectors/python.d.plugin/dovecot/metadata.yaml#L93-L110

Reply:
ah, Andy said the same thing, sorry for this

Reply:
Done